docs/multi-manager-setup.md

```diff
@@ -11,15 +11,15 @@ weight=3
 
 # High availability in Docker Swarm
```

Comment:
`--net=host` improves performences but I'm not sure we want to enforce it.

`<tls-config-flags>` I agree with alex, you need to mount them in the container, and it might confuse people if we explain this here.